ddtrace/internal/constants.py

```diff
@@ -0,0 +1,18 @@
+from ddtrace.constants import ANALYTICS_SAMPLE_RATE_KEY
+
+
+FILTERS_KEY = "FILTERS"
+SAMPLE_RATE_METRIC_KEY = "_sample_rate"
+SAMPLING_AGENT_DECISION = "_dd.agent_psr"
+SAMPLING_RULE_DECISION = "_dd.rule_psr"
+SAMPLING_LIMIT_DECISION = "_dd.limit_psr"
+HOSTNAME_KEY = "_dd.hostname"
+ENV_KEY = "env"
```

Comment:
this could be used publicly?

Reply:
There is no mention of this constants in the docs and, apart from `logging` the only other usage I can see are in internal modules. Based on these facts I drew the conclusion that this constant should perhaps be internal. Should I base my analysis on better criteria?

Reply:
I don't think I have any proof, just seems like a key customers could set on their own:

```python
ddtrace.config.tags = {ddtrace.constants.ENV_KEY: "prod}
```